ice_model.F90

```diff
@@ -593,12 +590,14 @@ subroutine ice_model_init( Ice, Time_Init, Time, Time_step_fast, Time_step_slow,
     !  -------- read restart --------
 !if (do_netcdf_restart) call ice_register_restart(Ice, 'ice_model.res.nc')
 
-if (file_exist('INPUT/ice_model.res.nc') ) then
+if (open_file(Ice_restart, "INPUT/ice_model.res.nc", 'read', Ice%domain)) then
   !if (mpp_pe() == mpp_root_pe()) call error_mesg ('ice_model_mod', &
   !         'Reading NetCDF formatted restart file: INPUT/ice_model.res.nc', NOTE)
-   call restore_state(Ice_restart)
```

Comment:
This didn't worked before and it still won't work now because the "albedo" variable is never registered so `restore_state` and `read_new_restart` are not going to do anything. Also we are "reading" the restart but not writing it, which defeats the purpose of the restarts. I think we should remove this whole block and just initialize the variables: 

https://github.com/NOAA-GFDL/ice_null/blob/d3aae9ac64c6b055990044fe3454613b6b8f4881/ice_model.F90#L655-L676